### ISSUE TITLE
Accuracy of model much worse than standard PyTorch inference

### ISSUE THREAD

**Issue by Rubi-Tomeu** [NVIDIA-AI-IOT/YOLOv5-with-Isaac-ROS#9]
Hello,

I trained a YOLOv5m model with a custom dataset (640x640px images in .jpg format), then tested it with unseen images and its accuracy was very good (confidence higher than 0.90, and detecting almost all targets).

I implemented your YOLOv5 Isaac ROS environment in a Jetson Orin AGX. Apparently, it works fine with pre-trained YOLOv5 models (Tested it real-time with an Alvium RGB camera). 

Then, I generated the TensorRT engine (.plan) from the ONNX file of my trained model, and performed inference real-time. However, now the accuracy in the detection is not so good. Most of the time it does not detect anything when the object is there in the image, and when it detects the objects, even the bbox are more or less ok, the confidence is very low.

The weird thing is that **the images that I used to train the network are exactly the same that I'm using now to test the model in the Isaac ROS environment** (extracted from a .bagfile). Therefore, I really don't what I'm doing wrong, maybe some problem while encoding the input tensor of the images? Any idea?

Thank you in advance.
